site/src/scaffold/index.ts

```diff
@@ -1,6 +1,6 @@
 import { handleCodeFilters } from './scripts/code-filter';
 import { handleFigmaFullScreenRequest } from './scripts/figma-embed';
-import { initTheme } from './scripts/theming';
+import { initTheme, ThemeTypeMap } from './scripts/theming';
```

Comment:
whoops, this one can be removed it looks like

Reply:
Removed